### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things I added to this repository
> Install react-router v5
Create my components and make them on one single page 
check for linter errors

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const catigories = () => (
+  <div>
```
Comment (elisha2kyakpo1) [inline src/components/Categories.js:2]: - [ ] kindly change the component to have a button with a text 'check status'

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const Books = () => (
+  <div>
```
Comment (elisha2kyakpo1) [inline src/components/Books.js:2]: - [ ] Good job adding this default component, from the instructions, but this component should have a button for removing a book, though not yet functional. You should also have placeholder data for books which should be made available to a child component through props.
- [ ] kindly create another component called book that will render books which it should receive through props, this is part of the project instructions.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @AlaaAlsalem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional 👍
- No linter errors 👍
- The readme file is well documented 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 2 above)
Comment (AlaaAlsalem) [reply src/components/Books.js:2]: Done, thanks for your feadback

File: src/components/Books.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    <h1> this is books</h1>
+    <button type="button">Remove a book</button>
+    <section className="book_list" />
+  </div>
```
Comment (AnselemOdims) [inline src/components/Books.js:5]: - [x] As pointed out by the previous reviewer, please create another component that will be a single book component.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @AlaaAlsalem , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Books.js (hunk 3 above)
Comment (AlaaAlsalem) [reply src/components/Books.js:5]: Done, Thanks for your feadback

**Review by AnselemOdims — APPROVED** [review]
### Hi  @AlaaAlsalem 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**